cla-backend-go/cmd/functional_tests/approval_list/approval_list.go

```diff
@@ -33,32 +29,129 @@ const (
 	claProspectiveManagerLFID = "cladevfunctionaltestuser"
 	//claProspectiveManagerUserName = "CLA Functional Test User Linux Foundation"
 	claProspectiveManagerEmail = "ddeal+cla+dev+functional+test+user@linuxfoundation.org"
-	claProjectSFID             = "a0941000005ouJFAAY"
-	claManagerCompanySFID      = "external-ee965ea2-ca83-4482-8a1b-94a468d9dcfa"
-	claGroupID                 = "d5412846-5dda-4c58-8f62-4c111a3cd0d3"
-	testGitHubUsername         = "dealako"
-	testDomainName             = "dealako.com"
-	testGitHubOrg              = "deal-test-org"
+
+	projectSFID = "a092M00001If9v8QAB"
+	intelSFID   = "0014100000TdzA7AAJ"
```

Comment:
the intelSFID = "014100000TdzA7AAJ" is not associated with intel and should be "00117000015vpjXAAQ". Discovered this issue on troubleshooting with Nirav